### ISSUE TITLE
{Linked:Bug9923250} [Adaptive Cards> Designer]:  Image of Account News logo is not loading

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8096]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25247.1000)
Browser: Edge Dev (109.0.1495.2)

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Designer' tab through tab key and activate it.
3. Navigate to the 'Sample data Editor' and ' Card payload editor' and attach the provided JSON file.
4. Navigate to 'Preview mode' and activate it.
5. Navigate to 'Account News' tile 
6. Now check if the image is loading and able to view or not

**Actual Result:**
Image(logo) is not loading and unable to view.
 
**Expected Result:**
Image(logo) should load and able to see.

**User Impact:**
Users will get impacted if image is not loading and users are unable to view the image

**Guideline Reference:**
[Guideline 101 Recommended Assistive Technology Tools for MAS Testing of Microsoft Digital Properties_v3_(CELA).docx (sharepoint.com)](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7b64DD7C5C-AA62-4D53-886E-BCAEE5341C01%7d&file=Guideline%20101%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties_v3_%28CELA%29.docx&action=default&mobileredirect=true&DefaultItemOpen=1)

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![usable- image logo for acccount news is not loading](https://user-images.githubusercontent.com/95913748/205232622-066e3e58-2605-41a5-a085-f68a19a02ba6.png)

### Card JSON

```JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.5",
    "body": [
        {
            "type": "Container",
            "items": [
                {
                    "type": "Image",
                    "url": "https://www.bing.com/th?id=OVFT._0U49JmkGsocqeNXrne86S&pid=News",
                    "altText": "Image for article [object Object]"
                },
                {
                    "type": "ColumnSet",
                    "spacing": "None",
                    "columns": [
                        {
                            "type": "Column",
                            "width": "auto"
                        },
                        {
                            "type": "Column",
                            "spacing": "None",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "subType": "Title",
                                    "text": "Account news",
                                    "wrap": false,
                                    "maxLines": 1,
                                    "spacing": "None"
                                }
                            ],
                            "width": "stretch"
                        }
                    ]
                },
                {
                    "type": "TextBlock",
                    "subType": "BodyPrimary",
                    "text": "A Simple Metric To Help Avoid Walmart-Like Stocks In Your Portfolio",
                    "spacing": "None",
                    "wrap": true
                }
            ]
        }
    ],
    "actions": [
        {
            "id": "ac-button-0",
            "type": "Action.Submit",
            "title": "More stories",
            "data": {
                "_internalQuickViewId": "AccountNewsList"
            }
        }
    ]
}
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

The image url from the card JSON is to the image shown in the card. 

URL: https://www.bing.com/th?id=OVFT._0U49JmkGsocqeNXrne86S&pid=News
<img width="280" alt="image" src="https://user-images.githubusercontent.com/98650930/205311180-ae19f6fe-4895-45d2-ab4d-a68b4f86e56f.png">

The URL should be updated if there is supposed to be a different experience.

Let me know if you have any questions!

**Comment by jwoo-msft** [comment]
as mentioned by @anna-dingler; image is loaded..
